_includes/community/meshmates-program-summary.html

```diff
@@ -1,26 +1,54 @@
-<div>
-  <h3>Layer5 MeshMates</h3>
+<h3 class="black-text">Layer5 Meetings</h3>
 
-  <h6>An onboarding and mentoring program</h6>
-  <p class="indent">
-    The Layer5 community is growing at a tremendous rate. No matter the size, we know that connection is what makes the Layer5 community different from any other community you join. Whether it is your first Layer5 meeting or your tenth contribution, as community organizers, we want to make sure you have a great time and are able to take advantage of all of the 
-    amazing learning, contributing, and community that Layer5 has to offer. 
-  </p>  
-  <p class="indent">
-    In the Layer5 mentor program, we will match you with an 
-    awesome MeshMate so you have a connection and guide from the start.
-  </p>
+<table>
```

Comment:
This file is for the meshmates-program-summary and should contain that summary. A new file can be created to capture the table of community workgroup resources.